app/controllers/application.js

```diff
@@ -94,7 +94,7 @@ export default Ember.Controller.extend({
       var item = this.get('store').peekRecord(type, itemId);
       var ordersPackages = this.store.peekAll('orders_package');
       var orderPackageId;
-      if(this.get('draftOrder')){
+      if(this.get('draftOrder') && ordersPackages.getEach('package.id').includes(itemId)){
```

Comment:
@abulsayyad123 , can we use getEach("packageId") here??

Reply:
Yeah.. that is working and its good one. Added those changes. Thanks